ephios/extra/templates/extra/widgets/previous_comments.html

```diff
@@ -0,0 +1,10 @@
+{% load i18n %}
+<div>
+    {% for comment in comments %}
+        <p class="mt-0 mb-0 d-flex">
+            <span class="pe-1"><b>{{ comment.author }}</b></span>
+            <span class="flex-grow-1">{{ comment.text }}</span>
+            <span>{% blocktranslate trimmed with comment.created_at|timesince as time %}{{ time }} ago{% endblocktranslate %}</span>
```

Comment:
I tried to translate this to german and it said "vor 5 Tage, 23 Stunde" and I didn't like that. "5 Tage, 23 Stunden her" könnte eher gehen.

Reply:
klingt halt seeeehr informell

Reply:
Gefällt mir auch nicht... Wie ist das denn bei dem Meldeschluss gelöst?

Reply:
turns out django macht das ootb

Reply:
knew it! :p